.github/workflows/e2e.yml

```
@@ -58,6 +58,32 @@ jobs:
           npm run fmt-check
         working-directory: ./e2e-tests
 
+  sync-test:
+    needs: [build, lint]
+    runs-on:
+      group: laos
+      labels: ubuntu-16-cores
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup
+      - uses: ./.github/actions/cache
+        with:
+          cache-key: test
+
+      - name: Download LAOS binary
+        uses: actions/download-artifact@v4
+        with:
+          name: laos-binary
+          path: ./target/release
+          
+      - name: Give executable permissions to LAOS binary
+        run: |
+          chmod +x ./target/release/laos
+          
+      - name: Warp Sync
+        run: |
+          SKIP_WASM_BUILD=1 cargo test --features warp-sync-test --locked
```

This runs all tests in the repo, maybe we're only interested in this particular test

```suggestion
          SKIP_WASM_BUILD=1 cargo test --locked -p laos --features warp-sync-test sync_test::warp_sync
```

---

Removing my PR suggestion as it's the same as this one

---

Agree. We only want to run that test in this workflow